framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -2744,9 +2744,10 @@ void VulkanReplayConsumerBase::ModifyCreateInstanceInfo(
                 graphics::vulkan_struct_get_pnext<VkDebugUtilsMessengerCreateInfoEXT>(pnext_callback_info);
         }
 
-        create_state.messenger_create_info             = { VK_STRUCTURE_TYPE_DEBUG_UTILS_MESSENGER_CREATE_INFO_EXT };
-        create_state.messenger_create_info.pNext       = modified_create_info.pNext;
-        create_state.messenger_create_info.flags       = 0;
+        create_state.messenger_create_info       = { VK_STRUCTURE_TYPE_DEBUG_UTILS_MESSENGER_CREATE_INFO_EXT };
+        create_state.messenger_create_info.pNext = modified_create_info.pNext;
+        create_state.messenger_create_info.flags = 0;
+
```

Comment:
This might be the worst clang-format issue I've seen yet. I had to add a blank line here to keep the clang-format output stable between the whole-file formatting that my editor does and the diff formatting that check_code_style.py does.